src/lib/mongoBackend/mongoGetSubscriptions.cpp

```diff
@@ -173,7 +176,7 @@ void mongoListSubscriptions
   std::string                    conds = std::string(CSUB_CONDITIONS) + "." + CSUB_CONDITIONS_TYPE;
   BSONObj                        q     = BSON(conds << "ONCHANGE");
 
-  if (!collectionQuery(getSubscribeContextCollectionName(tenant), q, &cursor, &err))
+  if (!collectionRangedQuery(getSubscribeContextCollectionName(tenant), q, limit, offset, &cursor, &count, &err))
```

Comment:
We should order results. In order to do so, we could use _id (as discover operation does with documents in the registrations collection). Have a look to this fragment of code from registrationsQuery() function:

```
  auto_ptr<DBClientCursor>  cursor;
  Query                     query(queryBuilder.obj());
  query.sort(BSON("_id" << 1));

  LM_T(LmtPagination, ("Offset: %d, Limit: %d, Details: %s", offset, limit, (details == true)? "true" : "false"));

  if (!collectionRangedQuery(getRegistrationsCollectionName(tenant), query, limit, offset, &cursor, countP, err))
  {
    return false;
  }
```